org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/quickfix/MarkerResolutionGenerator.java

```diff
@@ -55,6 +56,9 @@
 
 	@Inject
 	private WorkbenchMarkerResolutionAdapter.Factory adapterFactory;
+	
+	@Inject
+	TextualMultiModificationWorkbenchMarkerResolutionAdapter.Factory xxxxxx;
```

Comment:
Is the `xxxxx` as variable name on purpose?

Reply:
done